### PR TITLE
[codex] Fix proactive Phase 1 pass handling

### PR DESCRIPTION
**Pull request by wehos**
## Summary
- Treat explicit Phase 1 PASS outputs as structured per-channel pass signals.
- Skip music and meme fallback fetches when Phase 1 explicitly passed, preventing those channels from leaking into Phase 2 with random content.
- Strengthen recent proactive chat prompt boundaries so repeated angles must become [PASS].
- Add regression tests for PASS parsing and recent-chat prompt boundaries.

## Root Cause
Phase 1 parsed PASS as an absence of keyword, but the music/meme fallback path interpreted missing keywords as a request for random fallback content. That let explicitly passed channels still provide material to Phase 2.

## Validation
- uv run pytest tests/unit/test_proactive_phase1_pass.py
- uv run pytest tests/unit/test_mini_game_invite.py tests/unit/test_music_played_through_reset.py

## 发布说明

* **改进**
  * 更新了多语种“最近主动对话”提示，明确要求在无新角度时输出`[PASS]`并避免主题/措辞/语气重复（中/英/日/韩/俄）。
  * 强化了流程中对“PASS”信号的识别与响应，避免在被标记为PASS的频道进行不必要的后续内容拉取。

* **测试**
  * 新增单元测试，覆盖显式PASS识别、关键词提取与边界格式化行为。

### CODE REVIEW

File: main_routers/system_router.py [hunk 1]
```diff
@@ -1568,6 +1568,11 @@ def _parse_web_screening_result(text: str) -> dict | None:
     return None
 
 
+def _phase1_text_is_pass(text: str) -> bool:
+    """Return True when a Phase 1 section explicitly says PASS."""
+    return bool(re.search(r'(?i)(?:^|[\s\[])\s*PASS\s*(?:[\]\s]|$)', text or ''))
```
[the remark this answers is not in the document]
Comment (wehos) [reply main_routers/system_router.py]: Good catch, accepted. The parser now only treats an entire line equal to PASS or [PASS] as an explicit channel pass, so keyword phrases like pass the dutchie stay valid. Added a regression test for both music and meme keyword cases and pushed 4816c763b.

[the remark this answers is not in the document]
Comment (wehos) [reply main_routers/system_router.py]: Accepted and tightened further. _phase1_text_is_pass now uses a whole-section 
e.fullmatch, so only a section that is exactly PASS or [PASS] is treated as a channel pass. I also added a regression test for keyword: pass the dutchie followed by a stray [PASS] template line, and pushed 615394d47.

**Comment by wehos** [comment]
Checked this review item against the current branch and I don't see the reported stray naked literal. \\	ests/unit/test_proactive_phase1_pass.py:88\\ is not a standalone \\88\\, and \\Select-String -Pattern '^88$','\\b88\\b'\\ returns no matches. I also reran \\uv run pytest tests/unit/test_proactive_phase1_pass.py\\: 5 passed. Treating this as a false positive; no code change needed.
